ext/pdo/tests/pdo_drivers_basic.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+Test normal operation of PDO::getAvailableDrivers / pdo_drivers
+--CREDITS--
+Amo Chohan <amo.chohan@gmail.com>
+--SKIPIF--
+<?php # vim:ft=php
+if (!extension_loaded('pdo')) die('skip');
+?>
+--FILE--
+<?php
+ob_start();
```

Comment:
Per our conversation: This test is both self-referential (phpinfo and getAvilableDrivers come from the same source hashtable), and it's brittle in a way that will fail on systems which have PDO built, but have no drivers.